src/content/docs/en/guides/fonts.mdx

```diff
@@ -83,42 +75,43 @@ The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts
     </PackageManagerTabs>
 
     :::tip
-    You’ll find the correct package name in the “Quick Installation” section of each font page on Fontsource’s website. It will start with `@fontsource/` followed by the name of the font.
+    You'll find the correct package name in the “Quick Installation” section of each font page on Fontsource's website. It will start with `@fontsource/` followed by the name of the font.
     :::
 
-3. Import the font package in the layout or component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
+1. Import the font package in the layout or component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
```

Comment:
```suggestion
1. Import the font package in the component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
```
I'm thinking since we are explicitly callingout layout components below, we can just say "component" here?